crates/chain/src/keychain.rs

```diff
@@ -132,29 +129,29 @@ pub struct LocalChangeSet<K, A> {
     /// [`LocalChain`]: local_chain::LocalChain
     pub chain_changeset: local_chain::ChangeSet,
 
-    /// Additions to [`IndexedTxGraph`].
+    /// ChangeSet to [`IndexedTxGraph`].
     ///
     /// [`IndexedTxGraph`]: crate::indexed_tx_graph::IndexedTxGraph
-    pub indexed_additions: IndexedAdditions<A, DerivationAdditions<K>>,
+    pub indexed_changeset: indexed_tx_graph::ChangeSet<A, ChangeSet<K>>,
```

Comment:
I suggest renaming the following:

- `chain_changeset` field here to just `chain`
- `indexed_changeset` field here to `indexed_tx_graph` 
- Rename `LocalChangeSet` to `WalletChangeSet`
- Rename `LocalUpdate` to `WalletUpdate`. 

Those last two are suggestions would set the stage a bit for https://github.com/bitcoindevkit/bdk/issues/1061 (I want to move both those types to `bdk/wallet/mod.rs`.